firewood/tests/v2api.rs

```diff
@@ -0,0 +1,69 @@
+use std::{error::Error, path::PathBuf, sync::Arc};
+
+use firewood::{
+    db::{BatchOp, Db as PersistedDb, DbConfig, DbError, WalConfig},
+    v2::api::{Db, DbView, Proposal},
+};
+
+#[tokio::test(flavor = "multi_thread")]
+async fn smoke() -> Result<(), Box<dyn Error>> {
+    let cfg = DbConfig::builder()
+        .meta_ncached_pages(1024)
+        .meta_ncached_files(128)
+        .payload_ncached_pages(1024)
+        .payload_ncached_files(128)
+        .payload_file_nbit(16)
+        .payload_regn_nbit(16)
+        .wal(
+            WalConfig::builder()
+                .file_nbit(15)
+                .block_nbit(8)
+                .max_revisions(10)
+                .build(),
+        )
+        .truncate(true)
+        .build();
+    let db = Arc::new(testdb(cfg).await?);
+    let empty_hash = db.root_hash().await?;
+    assert_ne!(empty_hash, [0; 32]);
```

Comment:
Do we not have a unit test somewhere for this?

Reply:
I think this belongs elsewhere, but shouldn't hold up PR

Reply:
This is an integration/smoke test for v2::api::root_hash, which isn't really tested elsewhere right now.

Reply:
It should be! We.can remove this once it is, though.